extensions/amp-timeago/validator-amp-timeago.protoascii

```diff
@@ -36,6 +36,10 @@ tags: {  # <amp-timeago>
     value_regex: "\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d(:[0-5]\\d(\\.\\d+)?)?Z"
   }
   attrs: { name: "locale" }
+  attrs: {
```

Comment:
Could you move this above "datetime"? We try to keep the attributes in alphabetical order. Thanks!